pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -1678,23 +1678,34 @@
     private int managedLedgerMaxAckQuorum = 5;
     @FieldContext(
         category = CATEGORY_STORAGE_ML,
+        dynamic = true,
         doc = "Amount of memory to use for caching data payload in managed ledger. \n\nThis"
             + " memory is allocated from JVM direct memory and it's shared across all the topics"
             + " running in the same broker. By default, uses 1/5th of available direct memory")
     private int managedLedgerCacheSizeMB = Math.max(64,
             (int) (io.netty.util.internal.PlatformDependent.maxDirectMemory() / 5 / (1024 * 1024)));
+
+    @FieldContext(
+            category = CATEGORY_STORAGE_ML,
+            dynamic = true,
+            doc = "Threshold to evicteis triggered"
+    )
+    private double evictionTriggerThresholdPercent = 0.98;
```

Comment:
Normally, we may need a PIP to add new config. And you should provide solid reason to make it configurable.

Reply:
When the size of the cache configuration is small, 0.98 is too large, and the configurable value is more flexible.

Of course I can also remove this configuration and keep it fixed at 0.98.

Reply:
Fixed!